engine/verification/fetcher/chunkconsumer/consumer.go

```diff
@@ -94,14 +90,6 @@ func (w *Worker) Notify(chunkID flow.Identifier) {
 	w.consumer.NotifyJobIsDone(jobID)
 }
 
-// FinishProcessing is for the worker's underneath engine to report a chunk
-// has been processed without knowing the job queue
-// it's a callback so that the worker can convert the chunk id into a job
-// id, and notify the consumer about a finished job with the
-type FinishProcessing interface {
-	Notify(chunkID flow.Identifier)
-}
-
```

Comment:
We have a duplicate of this interface named `module.ProcessingNotifier`. Hence, we remove this and replace it.